<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>1024</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Choose the place</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../rec/venv/logo_only.ico</normaloff>../rec/venv/logo_only.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>1024</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>1024</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btn3</zorder>
   <zorder>btn1</zorder>
   <zorder>btn2</zorder>
   <zorder>btn4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
